lib/ui5Framework/npm/Installer.js

```diff
@@ -1,29 +1,28 @@
 import path from "node:path";
+import {mkdirp} from "../../utils/fs.js";
 import fs from "graceful-fs";
 import {promisify} from "node:util";
 import Registry from "./Registry.js";
-import rimraf from "rimraf";
+import AbstractInstaller from "../AbstractInstaller.js";
+import _rimraf from "rimraf";
+const rimraf = promisify(_rimraf);
```

Comment:
rimraf already returns a promise with version 4, so no promisify is needed

Reply:
Thanks!